trulens_eval/trulens_eval/feedback/benchmark_frameworks/meta_evaluation_examples.ipynb

```diff
@@ -0,0 +1,305 @@
+    "For experiment tracking, we provide a suite automatic metric computation via Aggregator, "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevance feedback function\n",
+    "from trulens_eval.feedback import GroundTruthAgreement, GroundTruthAggregator\n",
+    "from trulens_eval import Tru\n",
+    "import numpy as np\n",
+    "\n",
+    "tru = Tru()\n",
+    "tru.reset_database()\n",
+    "\n",
+    "golden_set = [\n",
+    "    {\n",
+    "        \"query\": \"who are the Apple's competitors?\",\n",
+    "        \"response\": \"Apple competitors include Samsung, Google, and Microsoft.\",\n",
+    "        \"expected_score\": 1.0,  # groundtruth score annotated by human\n",
+    "    },\n",
+    "    {\n",
+    "        \"query\": \"what is the capital of France?\",\n",
+    "        \"response\": \"Paris is the capital of France.\",\n",
+    "        \"expected_score\": 1.0,\n",
+    "    },\n",
+    "    {\n",
+    "        \"query\": \"what is the capital of Spain?\",\n",
+    "        \"response\": \"I love going to Spain.\",\n",
+    "        \"expected_score\": 0,\n",
+    "    },\n",
+    "]\n",
+    "# Create a Feedback object using the numeric_difference method of the ground_truth object\n",
+    "ground_truth = GroundTruthAgreement(golden_set)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from trulens_eval.feedback import Cortex\n",
+    "\n",
+    "provider = Cortex(model_engine=\"mistral-large\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Tuple\n",
+    "\n",
+    "\n",
+    "# output is feedback_score\n",
+    "def context_relevance_ff_to_score(input, output, benchmark_params) -> float:\n",
+    "    return provider.context_relevance(\n",
+    "        question=input,\n",
+    "        context=output,\n",
+    "        temperature=benchmark_params[\"temperature\"],\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "# output is (feedback_score, confidence_score)\n",
+    "def context_relevance_ff_to_score_with_confidence(\n",
+    "    input, output, benchmark_params\n",
+    ") -> Tuple[float, float]:\n",
+    "    return provider.context_relevance_verb_confidence(\n",
+    "        question=input,\n",
+    "        context=output,\n",
+    "        temperature=benchmark_params[\"temperature\"],\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Collect all prompt and expected responses from the golden set and pass to GroundTruthAggregator as ground truth labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prompts = []\n",
+    "responses = []\n",
+    "for i in range(len(golden_set)):\n",
```

Comment:
I'm not sure why you're doing `true_labels` separately. Shouldn't it all just be like:
```
prompts = [entry["query"] for entry in golden_set]
responses = [entry["response"] for entry in golden_set]
true_labels = [entry["expected_score"] for entry in golden_set]
```

Reply:
fixed - thanks for catching